packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -28,6 +28,70 @@
 							v-model="knobs.description"
 						/>
 					</AccordionTab>
+					<AccordionTab header="Suggested Configurations">
+						<template #header>
+							<Button
+								v-if="documentId && model?.id"
```

Comment:
Should this be disabled instead of not there all together?
May help a user realize why they may want to add a document in this workflow

Reply:
good idea, added